libnetwork/drivers/bridge/errors.go

```diff
@@ -319,25 +245,16 @@ func (ipv4 *IPv4AddrNoMatchError) Error() string {
 	return fmt.Sprintf("bridge IPv4 (%s) does not match requested configuration %s", ipv4.IP, ipv4.CfgIP)
 }
 
-// BadRequest denotes the type of this error
-func (ipv4 *IPv4AddrNoMatchError) BadRequest() {}
-
 // IPv6AddrNoMatchError is returned when the bridge's IPv6 address does not match configured.
 type IPv6AddrNoMatchError net.IPNet
 
 func (ipv6 *IPv6AddrNoMatchError) Error() string {
 	return fmt.Sprintf("bridge IPv6 addresses do not match the expected bridge configuration %s", (*net.IPNet)(ipv6).String())
 }
 
-// BadRequest denotes the type of this error
-func (ipv6 *IPv6AddrNoMatchError) BadRequest() {}
-
 // InvalidLinkIPAddrError is returned when a link is configured to a container with an invalid ip address
 type InvalidLinkIPAddrError string
```

Comment:
I'm not entirely sure if we should keep this error around. It's only called here: https://github.com/moby/moby/blob/7baab8bd6c0674475f108f4cd1158d9ea30a62cb/libnetwork/drivers/bridge/link.go#L69-L76

I only kept it because it's called two times but I think I should either un-export it or simply replace it with a stock error like I did for `IPTableCfgError`. @thaJeztah Any thoughts?